### PR TITLE
Add type definition for bero package

### PR DESCRIPTION
**Pull request by silversonicaxel**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/bero/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
```
Comment (danger-public) [inline types/bero/tsconfig.json:2]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## bero ([<kbd>unpkg</kbd>](https://unpkg.com/browse/bero@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'bero'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ddf62c7ccdea5d29ac3bb77a826e8e03706a31aa
</p>

File: types/bero/tsconfig.json (hunk 1 above)
Comment (silversonicaxel) [reply types/bero/tsconfig.json]: Fixed as suggested

File: types/bero/tslint.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "mode": "code",
+        "errors":[
+          [
+            "NeedsExportEquals",
+            false
+          ]
+        ]
+      }
+    ]
+  }
```
Comment (orta) [inline types/bero/tslint.json:16]: Are the two overrides here a copy & paste error?

Comment (silversonicaxel) [reply types/bero/tslint.json:16]: @orta it is not in progress, for me it is the final version, possibly to be merged

Comment (orta) [reply types/bero/tslint.json:16]: 👍🏻  sure

**Comment by silversonicaxel** [comment]
nice! thanks!

**Pull request merged**